Content.Client/Smoking/BurnStateVisualsComponent.cs

```diff
@@ -0,0 +1,13 @@
+namespace Content.Client.Smoking;
+
+    [RegisterComponent]
+    public sealed class BurnStateVisualsComponent : Component
+    {
+        [DataField("burntIcon")]
+        public string _burntIcon = "burnt-icon";
```

Comment:
turn these from the underscore names to something like “BurntIcon”, the underscore names are meant for private fields but if they’re public they need a naming scheme similar to other public classes